packages/loader/container-loader/src/container.ts

```diff
@@ -606,16 +605,16 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
                 protocolHandler: () => this._protocolHandler,
                 logConnectionStateChangeTelemetry: (value, oldState, reason) =>
                     this.logConnectionStateChangeTelemetry(value, oldState, reason),
-                propagateConnectionState: () => this.propagateConnectionState(),
-                isContainerLoaded: () => this.loaded,
                 shouldClientJoinWrite: () => this._deltaManager.shouldJoinWrite(),
                 maxClientLeaveWaitTime: this.loader.services.options.maxClientLeaveWaitTime,
             },
             this.logger,
         );
 
-        this.connectionStateHandler.on(connectEventName, (opsBehind?: number) => {
-            this.emit(connectEventName, opsBehind);
```

Comment:
So with this change container is not emitting "connect" event anymore. There might be some listening to this event?

Reply:
If removing this seems fine, then maybe we should add this in  BReaking.md?

Reply:
I'll add a note to BREAKING.md just in case.  The trivial usage in the gateway sample was the only instance I found across our repo plus Bohemia's -- I think it's likely not used since it doesn't represent a safe-to-use state for general purpose.

Reply:
Yeah, adding a note should be enough here.